README.md

```diff
@@ -75,6 +75,13 @@ To install with pip:
 pip install datadreamer
 ```
 
+Docker build and run:
+
+```bash
+docker build -t datadreamer .
+docker run -v "$(pwd):/app" datadreamer --save_dir generated_dataset
```

Comment:
On my laptop I also have to add `--runtime nvidia`, this might be good to mention as a sidenote.

Reply:
Can we also add it without build (to use from GHCR directly)?